src/components/PollList/pollList.js

```diff
@@ -7,34 +7,47 @@ import { Trash, PencilSquare } from "react-bootstrap-icons";
 import pollList from "../../redux/pollList/actions/pollList";
 import Header from "../Header/header";
 import AddPoll from "../AddPoll/addPoll";
+import voteCount from "../../redux/voteCount/actions/votecount";
+import SuccessMessage from "../../utils/successMessage/successMessage";
+import { emptyVoteCountSuccessStatus } from "../../redux/voteCount/actions/votecount";
+import { optionVoteCount } from "../../utils/voteCountUtils";
 
 const PollList = () => {
   const dispatch = useDispatch();
   const navigate = useNavigate();
-  const [pollIDs, setPollIDs] = useState([]);
+  const [showVoteCountSuccessMessage, setShowVoteCountSuccessMessage] =
+    useState(false);
+  const [pollOptionIds, setPollOptionIds] = useState({ pollIds: [], optionIds: [] });
   const [pageNumberLimit, setPageNumberLimit] = useState({
     pageNumber: 1,
     limit: 4,
   });
   const pollQuestion = useSelector((state) => state.pollList);
   const userDetails = useSelector((state) => state.login.userLogin);
+  const voteCountSuccessStatus = useSelector((state) => state.voteCount.status);
 
   useEffect(() => {
-    setPollIDs(JSON.parse(localStorage.getItem("pollIDs")) ?? []);
+    voteCountSuccessStatus === 200 && setShowVoteCountSuccessMessage(true);
```

Comment:
voteCountSuccessStatus === 200 @PuniaMohit This condition should be in Action, not in a component

Reply:
@meanstack9716  But sir if this condition satisfies, setShowVoteCountSuccessMessage(true) this function run which is usestate, but usestate cannot be used in action